### ISSUE TITLE
Adapt "About ECAM" Page

### ISSUE THREAD

**Issue by ElaChe00** [icra/ecam#305]
Hey Lluis, currently, the "About" page looks like this: 
![image](https://user-images.githubusercontent.com/45791540/107645851-8a2d4f80-6c79-11eb-8fcf-e4c3d0cf17be.png)

**Could you ple<se adapt the page to the example below?** 
I will share the files and icons via email.

![image](https://user-images.githubusercontent.com/45791540/107657609-98816880-6c85-11eb-809f-defaea45eb9c.png)

**Text blocks are:**

The “Energy Performance and Carbon Emissions Assessment and Monitoring Tool” (ECAM) offers unique capabilities for assessing greenhouse gas emissions and energy consumption at a system-wide level. Gain greater insights by identifying areas to reduce greenhouse gas emissions, increase energy savings and improve overall efficiencies to reduce costs. 

ECAM was developed by the Catalan Institute for Water Research ([ICRA](https://www.icra.cat/index.php?lang=3)) within the scope of the project “Water and Wastewater Companies for Climate Mitigation” ([WaCCliM](www.wacclim.org)), a joint initiative between the Deutsche Gesellschaft für Internationale Zusammenarbeit ([GIZ](http://www.giz.de/)) GmbH and the International Water Association ([IWA](https://iwa-network.org/)). 
WaCCliM is part of the International Climate Initiative ([IKI](https://www.international-climate-initiative.com/en/?iki_lang=en)). The German Federal Ministry for the Environment, Nature Conservation and Nuclear Safety ([BMU](https://www.bmu.de/en/)) supports this initiative on the basis of a decision adopted by the German Bundestag.

The first version of this tool was developed in 2015 as an Excel tool by the consortium Urban Water Commons (LNEC and ITA, Universitat Politècnica de València) in collaboration with Cobalt Water Global. The Excel version laid the foundation and basic equations for the web-tool.

ECAM 3.0 is free and open source (2015 -2021 license)

_____
Based on IPCC

ECAM was developed to be consistent with the IPCC Guidelines for National Greenhouse Gas Inventories and peer-reviewed literature.

_____
Free and Open Source

ECAM can be freely used, copied or changed. Its source code is openly available on GitHub. We encourage people to improve or make suggestions on how to improve the tool.
______
Secure

ECAM is secure and trustworthy. No information is stored on servers. All data inserted and processed during the ECAM assessment are merely on your personal computer.

_____

ECAM by IWA and GIZ, implemented by ICRA for WaCCliM Project* is licensed under a Creative Commons Attribution-ShareAlike 4.0 International License. Based on a work at www.wacclim.org. Permissions beyond the scope of this license may be available at wacclim@giz.de

**Comment by ElaChe00** [comment]
SVG Files [ECAM AboutPage Graphics.zip](https://github.com/icra/ecam/files/5966688/ECAM.AboutPage.Graphics.zip)

**Comment by ElaChe00** [comment]
[ECAM3.0_AboutPage_Header2.svg.zip](https://github.com/icra/ecam/files/5973022/ECAM3.0_AboutPage_Header2.svg.zip)

ECAM Header without text.

**Comment by ElaChe00** [comment]
[ECAM 3.0 AboutIcons.zip](https://github.com/icra/ecam/files/5973080/ECAM.3.0.AboutIcons.zip)

About Page Icons as SVG (Wrench, Statistic, Lock)

**Comment by holalluis** [comment]
done please check

**Comment by ElaChe00** [comment]
Could you add a padding to the left and right? Because now the whole text is strechted across the page:
![image](https://user-images.githubusercontent.com/45791540/108415823-49f83f00-722e-11eb-82c9-ae766339e2fe.png)

But it should look more like this:
![image](https://user-images.githubusercontent.com/45791540/108415797-44025e00-722e-11eb-889a-0e97e244ab79.png)

**Comment by holalluis** [comment]
done please check

**Comment by ElaChe00** [comment]
Yes! Additionally, is it possible to have the **links** within the text instead of separate? Like here:
![image](https://user-images.githubusercontent.com/45791540/108596523-dd905380-7385-11eb-852c-3e8e8bbd786a.png)

**Comment by holalluis** [comment]
The reason of putting the links below is because, having the translation in mind, I'm not sure how to insert the links inside each piece of  text for each language. Having them grouped in a list avoids having the code that renders the link inside the language json files. Can we put this on hold while I try to find a better solution?

**Comment by ElaChe00** [comment]
I see, no problem. We can solve this another time
